runtime/jvmti/jvmtiFunctionTable.c

```diff
@@ -63,7 +67,11 @@ jvmtiNativeInterface jvmtiFunctionTable = {
 	jvmtiRawMonitorNotifyAll,
 	jvmtiSetBreakpoint,
 	jvmtiClearBreakpoint,
+#if JAVA_SPEC_VERSION >= 9
 	jvmtiGetNamedModule,
+#else
+	NULL,
+#endif
```

Comment:
I suggest it would be worthwhile to add a helper macro:

```
#if JAVA_SPEC_VERSION >= 9
#define JVMTI_MODULE_ENTRY(name) name
#else /* JAVA_SPEC_VERSION >= 9 */
#define JVMTI_MODULE_ENTRY(name) NULL
#endif /* JAVA_SPEC_VERSION >= 9 */
```
Then use it here (and elsewhere) like this:
```
	JVMTI_MODULE_ENTRY(jvmtiGetNamedModule),
```
making it harder to mismatch the number of function references and NULLs.